staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1/types_jsonschema.go

```diff
@@ -16,328 +16,7 @@ limitations under the License.
 
 package v1
 
-// JSONSchemaProps is a JSON-Schema following Specification Draft 4 (http://json-schema.org/).
-type JSONSchemaProps struct {
```

Comment:
if we're aliasing, can we keep the v1 types the canonical types and alias things to them? that would make the diff a lot smaller, a lot easier to inspect for safety, and losing code history here would be unfortunate

Reply:
Let me try. Should work.

Reply:
Not so easy actually: v1 imports apiextensions. So we would have to put the JSONSchemaProp types into apiextensions. Let me see whether it makes it a little nicer.